thrift/src/main/thrift/rpc.thrift

```diff
@@ -238,6 +238,16 @@ struct TSDeleteDataReq {
     4: required i64 endTime
 }
 
+struct TSRawDataQueryReq {
+    1: required i64 sessionId
+    2: required list<string> paths
+    3: optional i32 fetchSize
+    4: required i64 startTime
+    5: required i64 endTime
+    6: required i64 statementId
```

Comment:
The statementId is generated by the server, not the client. So this field is not needed.

You could refer to generateQueryId(boolean isDataQuery) in TSServiceImpl.

Reply:
we need statementId to record queryId and release resource when close session.